lookout/core/analyzer.py

```diff
@@ -112,6 +112,16 @@ def train(cls, ptr: ReferencePointer, config: Mapping[str, Any],
         """
         raise NotImplementedError
 
+    @classmethod
```

Comment:
this is not part of the API so we should just implement it in format analyzer and not touch the core package